components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -1825,6 +1825,17 @@ public SubscribedAPI getSubscriptionById(int subscriptionId) throws APIManagemen
         return subscribedAPI;
     }
 
+    /** returns the SubscribedAPI object which is related to the UUID
+     *
+     * @param uuid UUID of Subscription
+     * @return
+     * @throws APIManagementException
+     */
+    public SubscribedAPI getSubscriptionByUUID(String uuid) throws APIManagementException {
```

Comment:
Please note this method was moved and already fixed in 59cfb8da4853453c203780ba4479fae32fbe603c